web/src/containers/Sidebar.js

```diff
@@ -206,7 +211,7 @@ class Sidebar extends Component {
                 {t('title', { year: '2018' })}
               </h1>
             </div>
-            <VerticalNav selectedId={activeSection} items={links} />
+            <VerticalNav selectedId={activeSection || "apd-state-profile-overview"} items={links} />
```

Comment:
With the new active item highlighting, there was no active item on page load, which caused a weird jump as the font bolded for the first time. That seems to be the only time there's no highlighted item, so this just sets the active item to the first section if `activeSection` is `null`.